src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -435,10 +441,11 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
     /**
      * Fetches the feed items for the sidebar
      *
+     * @param {boolean} shouldRefreshCache true if the cache should be refreshed
      * @param {boolean} shouldDestroy true if the api factory should be destroyed
      */
     fetchFeedItems(shouldRefreshCache: boolean = false, shouldDestroy: boolean = false) {
-        const { file, api, features } = this.props;
+        const { file, api, features, hasVersions: shouldShowVersions } = this.props;
```

Comment:
I was trying to stay consistent with the `hasDetails`, `hasSkills`, etc. props that ContentSidebar exposes, since these could also be considered top-level options rather than options we want to downplay. That said, we could easily go with `isFeatureEnabled(features, 'activityFeed.versions.enabled');` if that's preferable.

Reply:
Makes sense. I saw those as toggling the related sidebars in the `ContentSidebar` whereas the `features` affects what feed item types are available which is why it felt like a better fit. But either way is acceptable